src/components/SharedComponents/SubjectPhasePicker/SubjectPhasePicker.tsx

```diff
@@ -890,367 +667,410 @@ const SubjectPhasePicker: FC<SubjectPhasePickerData> = ({
-                    }
-                    $position="fixed"
-                    $bottom={0}
-                    $left={0}
-                    $right={0}
-                    $background="white"
-                    $height="100%"
-                    $overflowY="auto"
-                    $zIndex="modalDialog"
-                    $pa={24}
+                {/* DESKTOP PHASE PICKER */}
+                {showPhases && !isMobile && (
+                  <SelectionDropDownBox
+                    $background={"white"}
+                    $dropShadow="interactiveCardHover"
+                    $mt={8}
+                    $pa={28}
+                    $position="absolute"
+                    $top={"100%"}
+                    $zIndex={"modalDialog"}
+                    className="phase-selection"
                   >
-                    <OakFlex $flexDirection="column" $gap="space-between-m">
-                      <OakFlex
-                        $alignItems="center"
-                        $justifyContent="space-between"
+                    <FocusOn
+                      enabled={true}
+                      autoFocus={false}
+                      onClickOutside={() => setShowPhases(false)}
+                      onEscapeKey={() => setShowPhases(false)}
+                      scrollLock={false}
+                    >
+                      <FocusWrap
+                        onWrapStart={onFocusPhasesStart}
+                        onWrapEnd={onFocusPhasesEnd}
                       >
-                        <Button
-                          $ml={-8}
-                          size="large"
-                          label="Back"
-                          data-testid="mobile-phase-picker-back-to-subject-button"
-                          icon="chevron-left"
-                          $iconPosition="leading"
-                          variant="minimal"
-                          onClick={() => {
-                            setShowPhases(false);
-                            setIsMobileLotPickerModalOpen(true);
-                          }}
-                        />
-                        <CurriculumModalCloseButton
-                          ariaLabel="Close phase picker modal"
-                          onClose={() => setShowPhases(false)}
-                        />
-                      </OakFlex>
-
-                      <PhaseContainerWrapper>
-                        <OakHeading
-                          data-testid="mobile-phase-picker-heading"
-                          tag="h1"
-                          $font="heading-5"
-                        >
-                          School phase
-                        </OakHeading>
-
                         {showPhaseError && (
-                          <Flex id={phaseErrorId} $flexDirection="row" $mb={20}>
+                          <Flex
+                            id={phaseErrorId}
+                            role="alert"
+                            aria-live="polite"
```

Comment:
This is an old bit of code, but these two properties clash with each other - `role="alert"` basically means `aria-live="assertive"`. Also it won't actually do anything because announcements normally don't fire unless the element exists in the DOM before the content is updated.

Reply:
So both should be removed?

Reply:
I would suggest keeping the `role="alert"` because it is genuinely something people will want to know about immediately, but we should make a new ticket to get this looked at properly.